doc/client_lua_api.md

```diff
@@ -1114,6 +1124,30 @@ Can be obtained via `minetest.get_meta(pos)`.
 }
 ```
 
+### HUD Definition (`hud_add`, `hud_get`)
+```lua
+    {
+        hud_elem_type = "image", -- see HUD element types
```

Comment:
Defaults to `HUD_ELEM_TEXT` -> `"text"`
EDIT: Wrong defaults in position, scale, number (needs more docs too), item, size and undocumented world_pos (waypoint).

Reply:
I copied the example from the server lua api doc. I think it was meant to just be an example not a list of default values.